store/getters.js

```diff
@@ -480,6 +480,12 @@ function isLetterNumberOrHyphens (name, allowNone = false) {
     : allowNone
 }
 
+function isLetterNumberDotsOrUnderscores (name, allowNone = false) {
+  return name
+    ? name.length > 0 && /^[A-Za-z0-9_.]*$/.test(name)
```

Comment:
```suggestion
    ? name.length > 0 && /^\w+_?\d{2,}.?\d?$/.test(name)
```
This regex is stricter than the previous one, and still conforms to every reference ID I could find info about.

Reply:
Awesome, used it slightly modified :)